update.sh/target-pipeline.groovy

```diff
@@ -147,7 +147,7 @@ node {
 
 					git reset HEAD # just to be extra safe/careful
 					for dir in $dirs; do
-						git add "$dir/Dockerfile"* || true
+						git add "$dir" || true
```

Comment:
One problem is that this will be inaccurate for repos like redmine (https://github.com/docker-library/redmine), where there's a subdirectory with a `Dockerfile` that has slightly separate versioning (passenger version instead of redmine version), so the commits should be separate or the passenger versioning is lost.

In the cases where this is happening, is `generate-stackbrew-library.sh` actually including the new directories before they're committed? (since that's where this `$dirs` list comes from)